dev/tasks/linux-packages/apt/debian-buster/Dockerfile

```diff
@@ -68,6 +67,7 @@ RUN \
     python3-pip \
     rapidjson-dev \
     tzdata && \
+  if [ "$(uname -m)" = "x86_64" -o "$(uname -m)" = "i386"  ]; then apt install -y nvidia-cuda-toolkit;fi && \
```

Comment:
We use separated Dockerfile for arm64 because base image is different. (`debian:buster` and `arm64v8/debian:buster`.)
See `dev/tasks/linux-pakcages/debian-stretch-arm64` for details.

Reply:
You use 'debian-stretch-arm64' because you are crossbuilding for arm64 on x86-64. We use normal Dockerfiles when we build Arrow directly on arm64.

This change does not affect your x86-64 builds but helps us to build on arm64.

Reply:
```
./debian/pool/buster/main/a/apache-arrow:
apache-arrow_0.15.0~dev20190910-1.debian.tar.xz
apache-arrow_0.15.0~dev20190910-1.dsc
apache-arrow_0.15.0~dev20190910.orig.tar.gz
gir1.2-arrow-1.0_0.15.0~dev20190910-1_arm64.deb
gir1.2-gandiva-1.0_0.15.0~dev20190910-1_arm64.deb
gir1.2-parquet-1.0_0.15.0~dev20190910-1_arm64.deb
libarrow15_0.15.0~dev20190910-1_arm64.deb
libarrow15-dbgsym_0.15.0~dev20190910-1_arm64.deb
libarrow-dataset15_0.15.0~dev20190910-1_arm64.deb
libarrow-dataset15-dbgsym_0.15.0~dev20190910-1_arm64.deb
libarrow-dataset-dev_0.15.0~dev20190910-1_arm64.deb
libarrow-dev_0.15.0~dev20190910-1_arm64.deb
libarrow-flight15_0.15.0~dev20190910-1_arm64.deb
libarrow-flight15-dbgsym_0.15.0~dev20190910-1_arm64.deb
libarrow-flight-dev_0.15.0~dev20190910-1_arm64.deb
libarrow-glib15_0.15.0~dev20190910-1_arm64.deb
libarrow-glib15-dbgsym_0.15.0~dev20190910-1_arm64.deb
libarrow-glib-dev_0.15.0~dev20190910-1_arm64.deb
libarrow-glib-doc_0.15.0~dev20190910-1_all.deb
libarrow-python15_0.15.0~dev20190910-1_arm64.deb
libarrow-python15-dbgsym_0.15.0~dev20190910-1_arm64.deb
libarrow-python-dev_0.15.0~dev20190910-1_arm64.deb
libgandiva15_0.15.0~dev20190910-1_arm64.deb
libgandiva15-dbgsym_0.15.0~dev20190910-1_arm64.deb
libgandiva-dev_0.15.0~dev20190910-1_arm64.deb
libgandiva-glib15_0.15.0~dev20190910-1_arm64.deb
libgandiva-glib15-dbgsym_0.15.0~dev20190910-1_arm64.deb
libgandiva-glib-dev_0.15.0~dev20190910-1_arm64.deb
libgandiva-glib-doc_0.15.0~dev20190910-1_all.deb
libparquet15_0.15.0~dev20190910-1_arm64.deb
libparquet15-dbgsym_0.15.0~dev20190910-1_arm64.deb
libparquet-dev_0.15.0~dev20190910-1_arm64.deb
libparquet-glib15_0.15.0~dev20190910-1_arm64.deb
libparquet-glib15-dbgsym_0.15.0~dev20190910-1_arm64.deb
libparquet-glib-dev_0.15.0~dev20190910-1_arm64.deb
libparquet-glib-doc_0.15.0~dev20190910-1_all.deb

./debian/pool/stretch/main/a/apache-arrow:
apache-arrow_0.15.0~dev20190910-1.debian.tar.xz
apache-arrow_0.15.0~dev20190910-1.dsc
apache-arrow_0.15.0~dev20190910.orig.tar.gz
gir1.2-arrow-1.0_0.15.0~dev20190910-1_arm64.deb
gir1.2-gandiva-1.0_0.15.0~dev20190910-1_arm64.deb
gir1.2-parquet-1.0_0.15.0~dev20190910-1_arm64.deb
libarrow15_0.15.0~dev20190910-1_arm64.deb
libarrow15-dbgsym_0.15.0~dev20190910-1_arm64.deb
libarrow-dataset15_0.15.0~dev20190910-1_arm64.deb
libarrow-dataset15-dbgsym_0.15.0~dev20190910-1_arm64.deb
libarrow-dataset-dev_0.15.0~dev20190910-1_arm64.deb
libarrow-dev_0.15.0~dev20190910-1_arm64.deb
libarrow-flight15_0.15.0~dev20190910-1_arm64.deb
libarrow-flight15-dbgsym_0.15.0~dev20190910-1_arm64.deb
libarrow-flight-dev_0.15.0~dev20190910-1_arm64.deb
libarrow-glib15_0.15.0~dev20190910-1_arm64.deb
libarrow-glib15-dbgsym_0.15.0~dev20190910-1_arm64.deb
libarrow-glib-dev_0.15.0~dev20190910-1_arm64.deb
libarrow-glib-doc_0.15.0~dev20190910-1_all.deb
libarrow-python15_0.15.0~dev20190910-1_arm64.deb
libarrow-python15-dbgsym_0.15.0~dev20190910-1_arm64.deb
libarrow-python-dev_0.15.0~dev20190910-1_arm64.deb
libgandiva15_0.15.0~dev20190910-1_arm64.deb
libgandiva15-dbgsym_0.15.0~dev20190910-1_arm64.deb
libgandiva-dev_0.15.0~dev20190910-1_arm64.deb
libgandiva-glib15_0.15.0~dev20190910-1_arm64.deb
libgandiva-glib15-dbgsym_0.15.0~dev20190910-1_arm64.deb
libgandiva-glib-dev_0.15.0~dev20190910-1_arm64.deb
libgandiva-glib-doc_0.15.0~dev20190910-1_all.deb
libparquet15_0.15.0~dev20190910-1_arm64.deb
libparquet15-dbgsym_0.15.0~dev20190910-1_arm64.deb
libparquet-dev_0.15.0~dev20190910-1_arm64.deb
libparquet-glib15_0.15.0~dev20190910-1_arm64.deb
libparquet-glib15-dbgsym_0.15.0~dev20190910-1_arm64.deb
libparquet-glib-dev_0.15.0~dev20190910-1_arm64.deb
libparquet-glib-doc_0.15.0~dev20190910-1_all.deb

./ubuntu/pool/bionic/universe/a/apache-arrow:
apache-arrow_0.15.0~dev20190910-1.debian.tar.xz
apache-arrow_0.15.0~dev20190910-1.dsc
apache-arrow_0.15.0~dev20190910.orig.tar.gz
gir1.2-arrow-1.0_0.15.0~dev20190910-1_arm64.deb
gir1.2-gandiva-1.0_0.15.0~dev20190910-1_arm64.deb
gir1.2-parquet-1.0_0.15.0~dev20190910-1_arm64.deb
libarrow15_0.15.0~dev20190910-1_arm64.deb
libarrow-dataset15_0.15.0~dev20190910-1_arm64.deb
libarrow-dataset-dev_0.15.0~dev20190910-1_arm64.deb
libarrow-dev_0.15.0~dev20190910-1_arm64.deb
libarrow-flight15_0.15.0~dev20190910-1_arm64.deb
libarrow-flight-dev_0.15.0~dev20190910-1_arm64.deb
libarrow-glib15_0.15.0~dev20190910-1_arm64.deb
libarrow-glib-dev_0.15.0~dev20190910-1_arm64.deb
libarrow-glib-doc_0.15.0~dev20190910-1_all.deb
libarrow-python15_0.15.0~dev20190910-1_arm64.deb
libarrow-python-dev_0.15.0~dev20190910-1_arm64.deb
libgandiva15_0.15.0~dev20190910-1_arm64.deb
libgandiva-dev_0.15.0~dev20190910-1_arm64.deb
libgandiva-glib15_0.15.0~dev20190910-1_arm64.deb
libgandiva-glib-dev_0.15.0~dev20190910-1_arm64.deb
libgandiva-glib-doc_0.15.0~dev20190910-1_all.deb
libparquet15_0.15.0~dev20190910-1_arm64.deb
libparquet-dev_0.15.0~dev20190910-1_arm64.deb
libparquet-glib15_0.15.0~dev20190910-1_arm64.deb
libparquet-glib-dev_0.15.0~dev20190910-1_arm64.deb
libparquet-glib-doc_0.15.0~dev20190910-1_all.deb

./ubuntu/pool/cosmic/universe/a/apache-arrow:
apache-arrow_0.15.0~dev20190910-1.debian.tar.xz
apache-arrow_0.15.0~dev20190910-1.dsc
apache-arrow_0.15.0~dev20190910.orig.tar.gz
gir1.2-arrow-1.0_0.15.0~dev20190910-1_arm64.deb
gir1.2-gandiva-1.0_0.15.0~dev20190910-1_arm64.deb
gir1.2-parquet-1.0_0.15.0~dev20190910-1_arm64.deb
libarrow15_0.15.0~dev20190910-1_arm64.deb
libarrow-dataset15_0.15.0~dev20190910-1_arm64.deb
libarrow-dataset-dev_0.15.0~dev20190910-1_arm64.deb
libarrow-dev_0.15.0~dev20190910-1_arm64.deb
libarrow-flight15_0.15.0~dev20190910-1_arm64.deb
libarrow-flight-dev_0.15.0~dev20190910-1_arm64.deb
libarrow-glib15_0.15.0~dev20190910-1_arm64.deb
libarrow-glib-dev_0.15.0~dev20190910-1_arm64.deb
libarrow-glib-doc_0.15.0~dev20190910-1_all.deb
libarrow-python15_0.15.0~dev20190910-1_arm64.deb
libarrow-python-dev_0.15.0~dev20190910-1_arm64.deb
libgandiva15_0.15.0~dev20190910-1_arm64.deb
libgandiva-dev_0.15.0~dev20190910-1_arm64.deb
libgandiva-glib15_0.15.0~dev20190910-1_arm64.deb
libgandiva-glib-dev_0.15.0~dev20190910-1_arm64.deb
libgandiva-glib-doc_0.15.0~dev20190910-1_all.deb
libparquet15_0.15.0~dev20190910-1_arm64.deb
libparquet-dev_0.15.0~dev20190910-1_arm64.deb
libparquet-glib15_0.15.0~dev20190910-1_arm64.deb
libparquet-glib-dev_0.15.0~dev20190910-1_arm64.deb
libparquet-glib-doc_0.15.0~dev20190910-1_all.deb

./ubuntu/pool/disco/universe/a/apache-arrow:
apache-arrow_0.15.0~dev20190910-1.debian.tar.xz
apache-arrow_0.15.0~dev20190910-1.dsc
apache-arrow_0.15.0~dev20190910.orig.tar.gz
gir1.2-arrow-1.0_0.15.0~dev20190910-1_arm64.deb
gir1.2-gandiva-1.0_0.15.0~dev20190910-1_arm64.deb
gir1.2-parquet-1.0_0.15.0~dev20190910-1_arm64.deb
libarrow15_0.15.0~dev20190910-1_arm64.deb
libarrow-dataset15_0.15.0~dev20190910-1_arm64.deb
libarrow-dataset-dev_0.15.0~dev20190910-1_arm64.deb
libarrow-dev_0.15.0~dev20190910-1_arm64.deb
libarrow-flight15_0.15.0~dev20190910-1_arm64.deb
libarrow-flight-dev_0.15.0~dev20190910-1_arm64.deb
libarrow-glib15_0.15.0~dev20190910-1_arm64.deb
libarrow-glib-dev_0.15.0~dev20190910-1_arm64.deb
libarrow-glib-doc_0.15.0~dev20190910-1_all.deb
libarrow-python15_0.15.0~dev20190910-1_arm64.deb
libarrow-python-dev_0.15.0~dev20190910-1_arm64.deb
libgandiva15_0.15.0~dev20190910-1_arm64.deb
libgandiva-dev_0.15.0~dev20190910-1_arm64.deb
libgandiva-glib15_0.15.0~dev20190910-1_arm64.deb
libgandiva-glib-dev_0.15.0~dev20190910-1_arm64.deb
libgandiva-glib-doc_0.15.0~dev20190910-1_all.deb
libparquet15_0.15.0~dev20190910-1_arm64.deb
libparquet-dev_0.15.0~dev20190910-1_arm64.deb
libparquet-glib15_0.15.0~dev20190910-1_arm64.deb
libparquet-glib-dev_0.15.0~dev20190910-1_arm64.deb
libparquet-glib-doc_0.15.0~dev20190910-1_all.deb

./ubuntu/pool/xenial/universe/a/apache-arrow:
apache-arrow_0.15.0~dev20190910-1.debian.tar.xz
apache-arrow_0.15.0~dev20190910-1.dsc
apache-arrow_0.15.0~dev20190910.orig.tar.gz
gir1.2-arrow-1.0_0.15.0~dev20190910-1_arm64.deb
gir1.2-parquet-1.0_0.15.0~dev20190910-1_arm64.deb
libarrow15_0.15.0~dev20190910-1_arm64.deb
libarrow-dataset15_0.15.0~dev20190910-1_arm64.deb
libarrow-dataset-dev_0.15.0~dev20190910-1_arm64.deb
libarrow-dev_0.15.0~dev20190910-1_arm64.deb
libarrow-glib15_0.15.0~dev20190910-1_arm64.deb
libarrow-glib-dev_0.15.0~dev20190910-1_arm64.deb
libarrow-glib-doc_0.15.0~dev20190910-1_all.deb
libarrow-python15_0.15.0~dev20190910-1_arm64.deb
libarrow-python-dev_0.15.0~dev20190910-1_arm64.deb
libparquet15_0.15.0~dev20190910-1_arm64.deb
libparquet-dev_0.15.0~dev20190910-1_arm64.deb
libparquet-glib15_0.15.0~dev20190910-1_arm64.deb
libparquet-glib-dev_0.15.0~dev20190910-1_arm64.deb
libparquet-glib-doc_0.15.0~dev20190910-1_all.deb
```

Reply:
Can we use `arm64v8/debian` image on arm64 too?
Then we can use `arm64v8/debian` instead of `debian` for arm64.

If we use `arm64v8/debian`, we can use the same Dockerfile on amd64 and arm64. Then we can build .deb files for arm64 on Azure Pipelines.